cookbook/integrations/aws/athena/athena.py

```diff
@@ -0,0 +1,100 @@
+"""
+Athena Query
+############
+
+This example shows how to use a Flyte AthenaTask to execute a query.
+"""
+
+from flytekit import kwtypes, task, workflow
+from flytekit.types.schema import FlyteSchema
+from flytekitplugins.athena import AthenaConfig, AthenaTask
+
+# %%
+# This is the world's simplest query. Note that in order for registration to work properly, you'll need to give your
+# Athena task a name that's unique across your project/domain for your Flyte installation.
+athena_task_no_io = AthenaTask(
+    name="sql.athena.no_io",
+    inputs={},
+    query_template="""
+        select 1
+    """,
+    output_schema_type=None,
+    task_config=AthenaConfig(database="mnist"),
+)
+
+
+@workflow
+def no_io_wf():
+    return athena_task_no_io()
+
+
+# %%
+# Of course, in real world applications we are usually more interested in using Athena to query a dataset.
+# In this case we've populated our vaccinations table with the publicly available dataset
+# `here <https://www.kaggle.com/gpreda/covid-world-vaccination-progress>`__.
```

Comment:
is having two `_` at the end the right way? I've seen one but then I don't know much about RST so I could be wrong.

Reply:
I believe either is valid but looks like one _ is more common. updated